draft-law-moq-cat4moqt.md

```diff
@@ -112,103 +112,221 @@ The "moqt" claim is defined by the following CDDL:
 
 ~~~~~~~~~~~~~~~
 $$Claims-Set-Claims //= (moqt-label => moqt-value)
-moqt-label = XXX TODO - how do we register this?
-moqt-value = [ + moqt-object ]
+moqt-label = TBD_MOQT
+moqt-value = [ + moqt-scope ]
+moqt-scope = [ moqt-actions, moqt-ns-match, moqt-track-match ]
```

Comment:
Making a note here that I did not switch to a concatenated version. Unless ['example.com', 'bob'] is supposed to be semantically interchangeable with ['example.co', 'mbob'], I don't think we can safely concat them.